clang/lib/Driver/Driver.cpp

```diff
@@ -6553,6 +6634,20 @@ const ToolChain &Driver::getOffloadingDeviceToolChain(
                                                            HostTC, Args);
       break;
     }
+    case Action::OFK_SYCL:
+      // TODO: Add additional Arch values for Ahead of Time support for SYCL.
```

Comment:
Since AoT support doesn't exist in Clang yet, this TODO comment seems premature.